system/driver/desktop/window.go

```diff
@@ -223,6 +223,12 @@ func (w *Window) SetWinSize(sz image.Point) {
 	})
 }
 
+func (w *Window) SetSize(sz image.Point) {
```

Comment:
`SetSize` and `SetWinSize` both here and in `WindowMulti` should set `w.PixSize` and `w.WnSize` respectively.

Reply:
I believe those get set when the window resize event comes back through.